System/DataModule.cpp

```diff
@@ -116,6 +117,10 @@ namespace RTE {
 			}
 		} else if (propName == "IsFaction") {
 			reader >> m_IsFaction;
+			if (m_IsMarket) { m_IsFaction = false; }
+		} else if (propName == "IsMarket") {
+			reader >> m_IsMarket;
+			if (m_IsFaction && m_IsMarket) { m_IsFaction = false; }
```

Comment:
Can you not simplify this to match the above check? No need to check if IsFaction already, you're just flipping a bool.